python/setup-css.py

```diff
@@ -27,12 +27,15 @@ def run_tests(self):
         errno = pytest.main(self.pytest_args)
         sys.exit(errno)
 
+    
+
```

Comment:
Please remove these extra blank lines.  (There should be two blank lines, not four.)
```suggestion
```

Reply:
Sorry for the inconvenience, thanks for the feedback